### ISSUE TITLE
The IJBC's accuracy of magface that i trained is slightly lower than original paper

### ISSUE THREAD

**Issue by Jung-Jun-Uk** [IrvingMeng/MagFace#28]
Hi,

To use the baseline for my paper, I trained MagFace, but it is a little lower than the original paper.

I used MS1MV2, model: iresnet100, hyper-parameters: la=10, ua=110, lm=0.45, um=0.8, lg=35

I just ran the "run.sh" and the result is as below.

IJBB, magqlt=1
0.4210  0.9131  0.9463  0.9643  0.9759  0.9867

IJBC, magqlt=1
0.8754  0.9429  0.9611  0.9734  0.9829  0.9903

Also when I imported and used the MagFace Loss part in my code(detailed setup is a little different), I got similar results as above.

Are there any other best hyper-parameter settings or skills?

Did you train "magface_epoch_00025.pth" with the same settings as your code?

**Comment by IrvingMeng** [comment]
Hi,

TAR@FAR=1e-4 is the most accepted metric in IJBB/C while TAR@FAR=1e-6 can be biased by noise. Considering this, I think your results are a bit better than those in the paper (96.11 v.s. 95.97).  

"magface_epoch_00025" is trained with the same hyper-parameters. However, it's based on the private code base in our company as stated in the Readme. There are differences between the original implementation and the released version.

For your paper, I think it's okay to report results of your model or the released one, or both :)

Best,
Qiang

**Comment by Jung-Jun-Uk** [comment]
Thank you for your prompt reply!
